src/components/Card.vue

```diff
@@ -46,7 +46,7 @@ export default {
   display: flex;
   flex-direction: column;
   margin-bottom: 2rem;
-  width: 100%;
+  width: 99%; // compensated for the shadow
```

Comment:
Not so sure about it: if we keep 100%, the card's shadow comes out of the Loader overlay.

Reply:
To make sure I understand;
Is this a work around? Bcse card shadows render above overlay of set at 100?

Reply:
not above, but outside. The overlay is set as `absolute` and inset in the parent container. But the card shadows flow out of that container, so it is not covered by the overlay.